src/zrender.ts

```diff
@@ -117,7 +118,19 @@ class ZRender {
             ? false
             : opts.useDirtyRect;
 
-        const painter = new painterCtors[rendererType](dom, storage, opts, id);
+        const isDark = window.matchMedia && window.matchMedia('(prefers-color-scheme: dark)').matches;
```

Comment:
We need to ensure the window exists. Add `env.hasGlobalWindow` before `window.matchMedia` and I recommend moving this line to 124 to avoid unnecessary invoking.